app/controllers/supplementary/RepresentativeDetailsPageController.scala

```diff
@@ -46,14 +46,13 @@ class RepresentativeDetailsPageController @Inject()(
     customsCacheService
       .fetchAndGetEntry[RepresentativeDetails](supplementaryDeclarationCacheId, RepresentativeDetails.formId)
       .map {
-        case Some(data) => Ok(representative_details(appConfig, RepresentativeDetails.form().fill(data)))
-        case _          => Ok(representative_details(appConfig, RepresentativeDetails.form()))
+        case Some(data) => Ok(representative_details(appConfig, RepresentativeDetails.form.fill(data)))
+        case _          => Ok(representative_details(appConfig, RepresentativeDetails.form))
       }
   }
 
   def submitRepresentativeDetails(): Action[AnyContent] = authenticate.async { implicit request =>
-    RepresentativeDetails
-      .form()
+    RepresentativeDetails.form
```

Comment:
Based on scala style form should be in the new line, so I'm not sure that you run scalafmt :)